app/services/sms_service.rb

```diff
@@ -1,8 +1,12 @@
 class SmsService
-  def send_sms(phone_number, message)
+  def send_sms(contact_info, message, type)
+    address = contact_info[:email_address]
+    phone_number = contact_info[:phone_number].to_s
     message_data = serialize_message_data(phone_number, message)
     response = send_request(message_data)
-    JSON.parse(response.body)
+    response_body = JSON.parse(response.body)
+    OutgoingSmsCommunication.record(address, response_body['response'], type)
```

Comment:
Curious to know why you created a `record` method instead of using `create` like you did with the emails.

Reply:
(after we discussed this live we talked it over and I left comments on the model above)